src/main/java/org/reaktivity/nukleus/mqtt/internal/stream/MqttServerFactory.java

```diff
@@ -708,8 +748,8 @@ private MqttServer(
             this.affinity = affinity;
             this.budgetId = budgetId;
             this.decoder = decodePacketType;
-            this.subscribers = new HashMap<>();
-            this.publishers = new HashMap<>();
+            this.subscribers = new ConcurrentHashMap<>();
+            this.publishers = new ConcurrentHashMap<>();
```

Comment:
We are single threaded per reaktor core, so no need for concurrent data structures.